### DISCUSSION TITLE
MIDI 8x11 key matrix and two (4067 multiplexer)

### DISCUSSION THREAD

**Discussion by DrewMidi2022** [tttapa/Control-Surface#740] [Q&A]
Hi! newbie here. I managed to get the basics of the midi note/midi output logic.
I need help getting my 61keys keyboard with an 8x11 matrix splitted to two (4067 multiplexer)
8 for the inputs going to one of the multiplexer and 11 output in the other multiplexer.
I am very new to coding and I know that this library will do the job, I just can't do the coding for multiplexer that will merge with the
midi notes.

ie: code for identifying the inputs and outputs of the multiplexers / getting the dedicated midi notes for the dedicated key press.

any help is so much appreciated, my project is sitting for 3months now. this library is the closest and easy to edit as far as my research goes.
Thank you in advance.

**Comment by tttapa** [comment]
For future reference: https://forum.arduino.cc/t/midi-keyboard-with-2-multiplexer/985485/34
